### PR TITLE
[CCM-18035]: Added troubleshooting for when the secrets expire for a CCM SMP customer

### PR DESCRIPTION
**Pull request by joeyouss**
Thanks for contributing to the Harness Developer Hub! Our code owners will review your submission.

## Description

* Please describe your changes: 
* Jira/GitHub Issue numbers (if any): 18035
* Preview links/images (Internal contributors only): 

## PR lifecycle

We aim to merge PRs within one week or less, but delays happen sometimes.

If your PR is open longer than two weeks without any human activity, please tag a [code owner](https://github.com/harness/developer-hub/blob/main/.github/CODEOWNERS) in a comment.

PRs must meet these requirements to be merged:

- [ ] Successful preview build.
- [ ] Code owner review.
- [ ] No merge conflicts.
- [ ] Release notes/new features docs: Feature/version released to at least one prod environment.

### CODE REVIEW

**Comment by tuffacton** [comment]
Usually we get a nice netlify link for the draft, can we see that?

File: docs/cloud-cost-management/get-started/ccm-smp/aws-smp.md [hunk 1]
```diff
@@ -913,10 +913,21 @@ The following are some secrets from platform-service that you need to update:
    SMTP_USERNAME: <SMTP_USERNAME>
    ```
 
+
+
 :::info
 To increase TimescaleDB to 100Gi, run: `kubectl edit pvc wal-volume-harness-timescaledb-0 -n <namespace>`. Features like Recommendations and Anomalies within CCM services use it.
 :::
 
+### Troubleshooting
```
Comment (iamanmolm) [inline docs/cloud-cost-management/get-started/ccm-smp/aws-smp.md:922]: Lets move it at the bottom of the doc.

File: docs/cloud-cost-management/get-started/ccm-smp/aws-smp.md [hunk 2]
```diff
@@ -913,10 +913,21 @@ The following are some secrets from platform-service that you need to update:
...
+If in case the keys expire, the secrets will have to be set again. We recommend to `kubectl delete`` the following pods:
```
Comment (iamanmolm) [inline docs/cloud-cost-management/get-started/ccm-smp/aws-smp.md:923]: Once the secret is updated you don't need to delete the pod right away. First you would have to update the secrets in respective secret.yaml and then delete the pod.

File: docs/cloud-cost-management/get-started/ccm-smp/azure-smp.md [hunk 3]
```diff
@@ -386,6 +386,15 @@ After these secrets are implemented, please `kubectl delete` any pods related to
 Please ensure your Persistent Volumes related to TimescaleDB is at least 100Gi, run: `kubectl edit pvc wal-volume-harness-timescaledb-0 -n <namespace>`. These volumes are critical for capabilities like Recommendations and Anomalies within CCM.
 :::
 
+### Troubleshooting
```
Comment (iamanmolm) [inline docs/cloud-cost-management/get-started/ccm-smp/azure-smp.md:389]: Lets move it at the bottom of the doc.

File: docs/cloud-cost-management/get-started/ccm-smp/azure-smp.md [hunk 4]
```diff
@@ -386,6 +386,15 @@ After these secrets are implemented, please `kubectl delete` any pods related to
...
+If in case the K8s secrets expire, the secrets will have to be set again. We recommend to `kubectl delete`` the following pods:
```
Comment (iamanmolm) [inline docs/cloud-cost-management/get-started/ccm-smp/azure-smp.md:390]: Once the secret is updated you don't need to delete the pod right away. First you would have to update the secrets in respective secret.yaml and then delete the pod.

File: docs/cloud-cost-management/get-started/ccm-smp/gcp-smp.md [hunk 5]
```diff
@@ -344,6 +344,15 @@ Following are some secrets from platform-service that you will need to update:
 Run `kubectl edit pvc wal-volume-harness-timescaledb-0 -n <namespace>` and increase to 100Gi. It is used by recommendations and anomalies features within CCM services.
 :::
 
+### Troubleshooting
```
Comment (iamanmolm) [inline docs/cloud-cost-management/get-started/ccm-smp/gcp-smp.md:347]: Lets move it at the bottom of the doc.

File: docs/cloud-cost-management/get-started/ccm-smp/gcp-smp.md [hunk 6]
```diff
@@ -344,6 +344,15 @@ Following are some secrets from platform-service that you will need to update:
...
+If in case the K8s secrets expire, the secrets will have to be set again. We recommend to `kubectl delete` the following pods:
```
Comment (iamanmolm) [inline docs/cloud-cost-management/get-started/ccm-smp/gcp-smp.md:348]: Once the secret is updated you don't need to delete the pod right away. First you would have to update the secrets in respective secret.yaml and then delete the pod.

**Review by iamanmolm — CHANGES_REQUESTED** [review]

**Review by nikunj-badjatya — APPROVED** [review]

**Pull request merged**